sdv/single_table/base.py

```diff
@@ -176,6 +155,13 @@ def _validate_column(self, column):
 
         return errors
 
+    def _validate_synthesizer_specific_rules(self, data):
```

Comment:
Feels a bit too verbose, maybe with just `_validate` similar to what we do with `RDT._transform` and other `_` methods is good enough to specify that is 'synthesizer specific'.